### ISSUE TITLE
allow complete submenu

### ISSUE THREAD

**Issue by oliverfoster** [cgkineo/adapt-devtools#22]
handy to be able to complete an entire submenu

**Comment by chris-steele** [comment]
@oliverfoster I've had to dial back the changes to the model attributes that `complete menu` does due to runtime problems. It's essentially question views that throw errors due to missing `userAnswer`, `_selectedItems` values etc. Just setting `_isComplete` should be sufficient - this is all `spoor` does by default and these days plugins like `pageLevelProgress` and `trickle` monitor `_isComplete`. N.B. you might not have noticed this problem crop up on 404269 because when you enter the assessment (after using `complete menu`) the assessment resets all the question models before the views render.

I'm just about to push another change that we discussed - the ability to complete/reset whatever is under the mouse if you left click & hold, then hit `c` (complete) or `r` (reset) - so this can be entire pages via the menu, articles, blocks or individual components. So really easy to rapidly and selectively complete stuff.
